odxtools/utils.py

```diff
@@ -3,11 +3,15 @@
 import re
 from typing import TYPE_CHECKING, Any, Dict, Optional
 
+from typing_extensions import SupportsBytes
+
 if TYPE_CHECKING:
     from .database import Database
     from .diaglayers.diaglayer import DiagLayer
     from .snrefcontext import SnRefContext
 
+BytesTypes = (bytearray, bytes, SupportsBytes)
```

Comment:
it's probably better to change this to
```suggestion
BytesTypes = Union[bytearray, bytes, SupportsBytes]
```

Reply:
isinstance supports union starting from python 3.10

Reply:
do tuples work on 3.8?

Reply:
yes, https://docs.python.org/3.8/library/functions.html#isinstance

Reply:
great! let's merge...